### PR TITLE
Bugs when measuring backwards annually. 

### PR DESCRIPTION
**Pull request by thorn573**
All issues were remedied so a specimen measured forwards/backwards annually/sub annually presents its data the same way. 
i.e. meta data banner should display same start & end date. 

1) Entering a date in start point popout.
Date entered represents the year of the span the next point covers. When measuring backwards annually, entered date must be decreased by one.  

2) Meta data banner displaying correct date. 
When measuring backwards, start year must be increased by 1. If measuring backwards annually, both start and end must be increased by 1. Also, if all data cleared from view data window, meta data banner will reset. 

3) View data window display incorrect dates / points not placing correctly
Added more cases for point placement.

Tested on Google Chrome & FireFox

### CODE REVIEW

File: _site/leaflet-treering.js [hunk 1]
```diff
@@ -2930,7 +2936,13 @@ function CreatePoint(Lt) {
               $(document).keypress(e => {
                 var key = e.which || e.keyCode;
                 if (key === 13) {
-                  Lt.data.year = parseInt(document.getElementById('year_input').value);
+                  if (Lt.measurementOptions.forwardDirection == false && Lt.measurementOptions.subAnnual == false) {
+                    // must subtract one so newest measurment is consistent with measuring forward value
+                    // issue only applies to meauring backwwards annually
+                    Lt.data.year = parseInt(document.getElementById('year_input').value - 1);
```
Comment (cmcfadden) [inline _site/leaflet-treering.js:2942]: Right now you're subtracting 1 from the value, which will coalesce it to an int anyways, then wrapping all of the in parseInt(). I think you want parseInt(document.getElementById('year_input').value) - 1;

File: _site/leaflet-treering.js [hunk 2]
```diff
@@ -2930,7 +2936,13 @@ function CreatePoint(Lt) {
...
+                  } else  {
+                    Lt.data.year = parseInt(document.getElementById('year_input').value);
```
Comment (cmcfadden) [inline _site/leaflet-treering.js:2944]: same

File: leaflet-treering.js [hunk 3]
```diff
@@ -2936,7 +2936,13 @@ function CreatePoint(Lt) {
               $(document).keypress(e => {
                 var key = e.which || e.keyCode;
                 if (key === 13) {
-                  Lt.data.year = parseInt(document.getElementById('year_input').value);
+                  if (Lt.measurementOptions.forwardDirection == false && Lt.measurementOptions.subAnnual == false) {
+                    // must subtract one so newest measurment is consistent with measuring forward value
+                    // issue only applies to meauring backwwards annually
+                    Lt.data.year = parseInt(document.getElementById('year_input').value - 1);
```
Comment (cmcfadden) [inline leaflet-treering.js:2942]: same

**Comment by cmcfadden** [comment]
looks good!

**Comment by thorn573** [comment]
I'm confused. Is that issue in 3 spots or just 1? They are all on the same line, but 2 are in _site/leaf... & 1 is in leaf...

If I fix the issue in leaflet-treering.js, it will fix all of them?

**Comment by cmcfadden** [comment]
hah sorry I didn't see what was happening there. Just change it in the leaflet-treering file.

**Pull request merged**